pip_api/_installed_distributions.py

```diff
@@ -56,10 +57,12 @@ def _old_installed_distributions(local: bool):
     return ret
 
 
-def _new_installed_distributions(local: bool):
+def _new_installed_distributions(local: bool, path: Optional[str]):
     list_args = ["list", "-v", "--format=json"]
     if local:
         list_args.append("--local")
+    if path is not None:
+        list_args.extend(["--path", path])
     result = call(*list_args)
```

Comment:
I don't think we need to really do a version check here. Like with the `--local` flag, we can just let the `call` fail.

Reply:
Actually, an old `pip` version will just silently ignore the paths if we do it like this. Probably better to add a check.

Reply:
796be87